druid-derive/Cargo.toml

```diff
@@ -17,3 +17,4 @@ proc-macro2 = "1.0.9"
 
 [dev-dependencies]
 druid = { path = "../druid", version = "0.6.0" }
+float-cmp = "0.6.0"
```

Comment:
Can we avoid the `num-traits` dependency?
```suggestion
float-cmp = { version = "0.6.0", default-features = false }
```

Reply:
Sure, we don't need it for this PR.